pkg/cmd/pulumi/install.go

```diff
@@ -139,7 +143,20 @@ func newInstallCmd() *cobra.Command {
 						}
 					}
 
-					pctx.Diag.Infoerrf(diag.Message("", "%s installing"), label)
+					// If we don't have a version yet but we are installing try and call GetLatestVersion
+					// to fill it in.
+					if install.Version == nil {
+						logging.V(preparePluginVerboseLog).Infof(
```

Comment:
I think typically we just hardcode the log level.  I think I kind of prefer that over local constants as it makes it slightly easier to see at which level we're logging at the log line instead of looking at another variable (it might be nice to have this as global constants with a meaning somewhere, I think these log levels are somewhat arbitrary right now)